.gitignore

```diff
@@ -358,3 +358,5 @@ MigrationBackup/
 data.json
 
 *.xlsx
+
+GoogleLinks Mocks
```

Comment:
whats this for ?

Reply:
try not to have spaces in your file/folder names, yknow its a headache to refer to them progomatically
also,
u could technically jsut add them to your repo
also, folder name should be something like `mocks`

Reply:
stay consistent in your naming of variables, classes, files, folders, etc.